dime-research-standards/pillar-3-research-reproducibility/checklists/tex/reproducibility-package-guidance-note.tex

```diff
@@ -0,0 +1,109 @@
+
+	\bigskip
+
+	The goal of this review is to ensure the release of reproducible and code, data, and outputs; and therefore make the project more transparent. DIME Analytics typically completes the reproducibility checks within two weeks of submission of the reproducibility package by their respective project teams, and sends the package back with a well-commented reproducibility review document, highlighting all the problems that might break the code, constitute non-reproducible results, or create difficulties to make the code legible. For example, some of these can be:
+
+	\bigskip
+
+	\begin{itemize}
+		\setlength\itemsep{-0.1em}
+		\item Lack of specification of file or folder paths
+		\item Installation of necessary software or packages
+		\item Lack of version and matsize settings in Stata
+		\item Analytical outputs are created but they are different to how they are presented in the academic publication
+		\item Lack of comments in the code to follow through the operations it conducts
+	\end{itemize}
+
+	The main task of the review is to make sure that the code runs smoothly in a computer different than where it was developed, and that all the outputs recreated by it exactly match the research outputs shown in the working paper provided.	
+DIME Analytics follows specific tasks when conducting the computational reproducibility checks, which are outlined below.
+
+	\subsection{1. \, Receving the reproducibility package}
+
+	The project team will send a code review request via email to DIME Analytics by providing a reproducibility package in a shared OneDrive or Dropbox folder, a GitHub repository, or a compressed folder sent by email if the files size allows. If using a GitHub repository, please share only the code files through GitHub and send the input data files using another channel. The package should contain the items listed below.
+
+	\bigskip
+
+	\begin{itemize}
+		\setlength\itemsep{-0.1em}
+		\item \textbf{Code:} All scripts necessary to reproduce the analytical outputs of the paper, including a well-documented and well-commented Master Script that allows to perform a one-button run of the code.
+		\item \textbf{Data:} Fully de-identified, no access-restricted, and complete data necessary to run the master script that reproduces the outputs (can be raw, clean, or constructed) . Any exception where access to restricted or identified datasets is needed to reproduce code outputs will be handled on a case-by-case basis.
+		\item \textbf{Output:} All raw outputs used for the paper (tables and figures), as created by the master script.
```

Comment:
we always delete them before the first run of the code. Still, when an output is stable but different than the paper they're helpful because we can use them to:
1. verify that their raw outputs are the same as the paper. If not, this is a strong signal of version mismatch between code/data and paper
2. If the raw outputs are the same we can check the differences between them and our results using GitHub Desktop to see if some patterns emerge, like if all the discrepancies are in one column or row. It's easier to catch this in GH Desktop than manually

Reply:
in short, we don't need them strictly speaking but they provide useful additional info for the review